AutomatedTesting/Gem/PythonTests/smoke/test_RemoteConsole_CPULoadLevel_Works.py

```diff
@@ -0,0 +1,105 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+
+UI Apps: AutomatedTesting.GameLauncher
+Launch AutomatedTesting.GameLauncher with Simple level
+Test should run in both gpu and non gpu
+"""
+
+import pytest
+import psutil
+
+# Bail on the test if ly_test_tools doesn't exist.
+pytest.importorskip("ly_test_tools")
+import ly_test_tools.environment.waiter as waiter
+from ly_remote_console.remote_console_commands import RemoteConsole as RemoteConsole
+from ly_remote_console.remote_console_commands import (
+    send_command_and_expect_response as send_command_and_expect_response,
+)
+
+
+@pytest.mark.parametrize("launcher_platform", ["windows"])
+@pytest.mark.parametrize("project", ["AutomatedTesting"])
+@pytest.mark.parametrize("level", ["Simple"])
+@pytest.mark.SUITE_smoke
+class TestRemoteConsoleLoadLevelWorks(object):
+    @pytest.fixture
+    def remote_console_instance(self, request):
+        console = RemoteConsole()
+
+        def teardown():
+            if console.connected:
+                console.stop()
+
+        request.addfinalizer(teardown)
+
+        return console
+
+    def test_RemoteConsole_LoadLevel_Works(self, launcher, level, remote_console_instance, launcher_platform):
+        expected_lines = ['Level system is loading "Simple"']
+
+        self.launch_and_validate_results_launcher(launcher, level, remote_console_instance, expected_lines)
+
+    def launch_and_validate_results_launcher(
```

Comment:
Why not use this from editor_python_test_tools.hydra_test_utils? Looks like it was copied out from there and will just double needed maintenance in the future.

Reply:
Looks like the GPU version of the test does the same thing. so would be tripled maintenance.

Reply:
using import